yarn.lock

```diff
@@ -50,151 +50,159 @@
     source-map "^0.5.0"
 
 "@babel/core@>=7.2.2", "@babel/core@^7.1.0", "@babel/core@^7.1.6", "@babel/core@^7.4.5":
-  version "7.6.4"
```

Comment:
Do you know what happened with yarn?

Reply:
Yeah I kinda broke this and had to reinstall... i did yarn add jest, but that is already installed with create react app, so it caused mad hiccups. I had to torch node_modules and re yarn install. (that was the recommendation from yarn).